<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListaOperacoesWidget</class>
 <widget class="QDockWidget" name="ListaOperacoesWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icones/icones/pgsqlModeler48x48.png</normaloff>:/icones/icones/pgsqlModeler48x48.png</iconset>
  </property>
  <property name="allowedAreas">
   <set>Qt::AllDockWidgetAreas</set>
  </property>
  <property name="windowTitle">
   <string>Operações Executadas</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="0" column="0" colspan="3">
     <widget class="QTreeWidget" name="operacoes_tw">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Link">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>14</width>
        <height>14</height>
       </size>
      </property>
      <property name="indentation">
       <number>20</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="operacoes_lb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Operações:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="num_operacao_lb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="posicao_lb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Posição:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="num_posicao_lb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>178</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="excluiroperacoes_tb">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Excluir histórico de operações</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/icones/icones/excluir.png</normaloff>:/icones/icones/excluir.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="desfazer_tb">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Desfazer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/icones/icones/desfazer.png</normaloff>:/icones/icones/desfazer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="refazer_tb">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Refazer</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/icones/icones/refazer.png</normaloff>:/icones/icones/refazer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
